iso15118/secc/controller/simulator.py

```diff
@@ -578,6 +578,62 @@ async def get_evse_status(self) -> EVSEStatus:
     async def set_present_protocol_state(self, state_name: str):
         pass
 
+    async def send_charging_power_limits(
+        self,
+        protocol: Protocol,
+        control_mode: ControlMode,
+        selected__energy_service: ServiceV20,
+    ) -> None:
+        """
+        This method shall merge the EV-EVSE charging power limits and send it
+
+        Args:
+            protocol: protocol selected (DIN, ISO 15118-2, ISO 15118-20_AC,..)
+            control_mode: Control mode for this session - Scheduled/Dynamic
+            selected__energy_service: Enum for this Service - AC/AC_BPT/DC/DC_BPT
+
+        Returns: None
+
+        """
+        if protocol == Protocol.ISO_15118_20_AC:
+            if selected__energy_service in [ServiceV20.AC, ServiceV20.AC_BPT]:
+                charge_parameters = await self.get_ac_charge_params_v20(
+                    selected__energy_service
+                )
+            else:
+                charge_parameters = await self.get_dc_charge_params_v20(
+                    selected__energy_service
+                )
```

Comment:
Do we expect the dc to be hit? Maybe once we add support for DC...if so the if condition above for protocol check could be modified to include ISO_15118_20_DC as well...